test/hasSpaceBeforePrecompiler

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#
+# Copyright 2016 Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+# Search recursive inside our C++ source files if they violate our coding
+# style by using the wrong pre-compiler spacing style.
+# See:
+#   https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md
```

Comment:
Could you use the more explicit:
`https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#7-preprocessor` here?